org/mozilla/jss/ssl/javax/JSSParameters.java

```diff
@@ -0,0 +1,174 @@
+package org.mozilla.jss.ssl.javax;
+
+import javax.net.ssl.*;
+import java.util.*;
+
+import org.mozilla.jss.ssl.*;
+
+/**
+ * JSSParameters is an implementation of SSLParameters to interoperate
+ * with NSS.
+ *
+ * In particular, we extend SSLParameters to provide cipher suites and
+ * protocol versions as NSS constants. This aids with the JNI translation
+ * layer. We also create a default set of SSLParameters which use a null
+ * cipher suite list and null protocol version list to fall back on the
+ * NSS default. On RHEL-like systems, this will default to CryptoPolicies.
```

Comment:
Crypto Policies needs to be implemented; I think I'll make that a separate PR as it touches multiple parts of the new system.

(We can use @abbra's 1c1dad189073a9aa2fa5b5503c659967a25dc119 as a reference on the protocol side but there doesn't seem to be a corresponding commit for the SSLCipher side).